public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+App.controller('App.Ctrl', function($scope, $timeout, AppService) {
+    const colorCode = {1:"green", 2: "red", 3: "yellow", 4: "blue"};
```

Comment:
This is a personal preference for me, but I find it makes your code easier to navigate.  Any constants that won't change I usually name them with all aps and underscore styling.  Like COLOR_CODE, so its crystal clear anywhere in the code that this is a constant.  Also, I think its better to use COLOR_CODE.GREEN = 'green' instead of COLOR_CODE[1] = green since green should be green regardless of positioning.  Try to find a way to not bind your color constants with your app logic.